test/common_image_captioning.py

```diff
@@ -0,0 +1,278 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
please use `2018` in new files.

Reply:
Ok. I will double-check all the new license headers and fix it.